integration-tests/test/test_unbond.py

```diff
@@ -32,7 +32,9 @@ def get_total_cost_from_block(block_info: BlockInfo) -> int:
         total_cost += deploy.cost
     return total_cost
 
-
+# TODO https://github.com/rchain/rchain/issues/3572
+# pylint: disable=too-many-locals
+@pytest.mark.skip
```

Comment:
https://github.com/rchain/rchain/issues/3572
There are something wrong in unbonding.

Reply:
Is the missing block from slashed validator?

Reply:
The missing block is from unbonded validator.